proposals/implicit-lambda-parameters-with-modifiers.md

```diff
@@ -30,25 +30,32 @@ implicit_anonymous_function_signature
     ;
 
 implicit_anonymous_function_parameter_list
--    : implicit_anonymous_function_parameter (',' implicit_anonymous_function_parameter)*
-+    : implicit_anonymous_function_parameter_ex (',' implicit_anonymous_function_parameter_ex)*
+    : implicit_anonymous_function_parameter (',' implicit_anonymous_function_parameter)*
     ;
 
 implicit_anonymous_function_parameter
-    : identifier
+-   : identifier
++   : 'scoped'? anonymous_function_parameter_modifier? identifier
     ;
 
-implicit_anonymous_function_parameter_ex
-    : anonymous_function_parameter_modifier? identifier
+explicit_anonymous_function_parameter
+    : 'scoped'? anonymous_function_parameter_modifier? type identifier
+    ;
+
+anonymous_function_parameter_modifier
+    : 'in'
+    | 'ref' 'readonly'?
```

Comment:
I'm not able to find where `readonly` started being valid syntax within a lambda parameter declaration. <https://github.com/dotnet/csharplang/blob/main/proposals/csharp-12.0/ref-readonly-parameters.md> says "No changes in grammar are necessary," but searching the repo for `'readonly'` as well as for `anonymous_function_parameter_modifier` failed to show where the the grammar had been updated.

Reply:
Well, `ref readonly` was added in C# 7: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/variables#971-general

That may be what this means

Reply:
So perhaps the C# 12 "ref readonly parameters" proposal overlooked adding `'readonly'` to `anonymous_function_parameter_modifier`?